tox.ini

```diff
@@ -17,39 +17,39 @@ whitelist_externals =
 
 [testenv:es]
 commands =
-    make -f Makefile-docker update_deps
+    make -f Makefile-docker install_python_test_dependencies
     pytest -m "es_tests and not needs_locales_compilation" --ignore=tests/ui/ -v {posargs}
 
 [testenv:addons]
 commands =
-    make -f Makefile-docker update_deps
+    make -f Makefile-docker install_python_test_dependencies
     pytest -n 2 -m 'not es_tests and not needs_locales_compilation' -v src/olympia/addons/ {posargs}
 
 [testenv:devhub]
 commands =
-    make -f Makefile-docker update_deps
+    make -f Makefile-docker install_python_test_dependencies
```

Comment:
I'm actually a bit confused that not installing node dependencies still makes the tests succeed. I fear we're running into a scenario where the whole test-suite is relying on the cached `node_modules` folder. Let's clear the caches and see what fails?

Reply:
Clearing everything and seeing what failed was how I developed this 😆   So I'll be interested to see what fails too.

Reply:
Ah, now that makes sense :)

Reply:
Well, the failures do. Not sure how it worked before. Let's blame caching.